package.json

```diff
@@ -26,6 +27,7 @@
     "moment": "^2.29.1",
     "moment-timezone": "^0.5.33",
     "react": "^17.0.2",
+    "react-advanced-form": "^1.7.2",
```

Comment:
Similar to my other comments I think adding validation is not within the scope of this release cycle. I would also be hesitant to add another dependency if it isn't necessary.

Reply:
Thanks Mychael for your valuable feedback. As per the requirement, we had to add popups and there was the need of adding material-ui library to do the same. Also validation was required because, there is a functionality of button disable/enable depending on the form validation.